Makefile.am

```diff
@@ -638,9 +638,13 @@ tests_test_security_curve_SOURCES += \
 endif
 
 tests_test_security_curve_LDADD = \
-	src/libzmq.la
+	src/libzmq.la $(LIBUNWIND_LIBS)
```

Comment:
It's ifdef'd in err.hpp - which is included by the test, so it was breaking when the library was there because the test was not being linked against it, despite the ifdef being true